DB/src/main/java/io/deephaven/db/v2/sources/chunk/BooleanChunk.java

```diff
@@ -14,11 +14,6 @@
 // region BufferImports
 // endregion BufferImports
 
-/**
- * {@link Chunk} implementation for boolean data.
- *
- * @IncludeAll
- */
```

Comment:
I don't think we should be getting rid of this javadoc - `@IncludeAll` is a special tag. I'm not sure if I explicitly deleted this in my original branch or not, but we should probably keep it. (This goes for the other chunk javadocs too.)

Reply:
the javadoc comments were most likely added after your commit. Will restore.